plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -62,6 +63,7 @@
 import static io.trino.spi.type.RealType.REAL;
 import static io.trino.spi.type.SmallintType.SMALLINT;
 import static io.trino.spi.type.TimeType.TIME;
+import static io.trino.spi.type.TimestampType.MAX_PRECISION;
```

Comment:
This shouldn't be static imported, as "max precision" is not descriptive in the code dealing with _various_ types.

i know it's pre-existing for MAX_SHORT_PRECISION, but let's make the new one better, or fix both.

Reply:
Just to confirm, you're asking me to rename MAX_PRECISION to MAX_LONG_TIMESTAMP_PRECISION in the SPI module?

Reply:
No. To use qualified name here, without static import.

Reply:
Done!